mysql-test/suite/sys_vars/r/wsrep_sst_auth_basic.result

```diff
@@ -1,8 +1,6 @@
 #
 # wsrep_sst_auth
 #
-# save the initial value
-SET @wsrep_sst_auth_global_saved = @@global.wsrep_sst_auth;
 # default
```

Comment:
unclear why the save and restore of global variables is being removed from these tests. The masking reason isn't sufficient, MTR self checking ensures that the post test values are the same as previous.

Reply:
These are removed for two reasons:
1. they are not needed (as one can see from the results) - they were introduced when the value was not masked and plaintext, thus back then we had to restore the text.
2.  they operate under the wrong assumption that we can recover an original value of  `@@global.wsrep_sst_auth`  - no, all we can recover is a mask `********` that has nothing to do with the original value.